theme/ckeditor5-ui/components/formheader/formheader.css

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+:root {
+	--ck-table-form-header-height: 38px;
```

Comment:
This is no longer a table component but a generic one. This should be more like `--ck-form-header-height`.